javascript/javascript1/week4/voiceassistant.js

```diff
@@ -0,0 +1,54 @@
+let toDoList = [];
+
+function getToday() {
+  let today = new Date();
+  let day = today.getDate();
+  let month = today.getMonth() + 1;
+  let year = today.getFullYear();
+  let marr = new Array(
+    "January",
+    "February",
+    "March",
+    "April",
+    "May",
+    "June",
+    "July",
+    "August",
+    "September",
+    "October",
+    "November",
+    "December"
+  );
+  console.log(day + " of " + marr[month] + " " + year);
+}
+
+function myFunction() {
+  console.log("Timer Done");
+}
+
+function getReply(command) {
+  if (command == "Hello my name is Benjamin") {
+    console.log("nice to meet you Benjamin");
```

Comment:
What if user entered command is in lower case or upper case?  When you are working  with string should always convert strings to lower case before comparing.

What if user command includes other text than "hello my name is ".  in that case  '=='  operator returns false, In this case I suggest using include().

In this case you are calling getReplay(), so it should return the output string, instead of console.log.

Try to make greeting dynamic by taking the name from command.